apps/tup-cms/src/taccsite_custom/tup_cms/static/tup_cms/css/tup-cms.for-core-cms.css

```diff
@@ -16,7 +16,7 @@
 /* COMPONENTS */
 @import url("./for-core-cms/components/c-footer.css") layer(project);
 @import url("./for-core-cms/components/django.cms.blog.app.page.css") layer(project);
-@import url("./for-core-cms/components/lightgallery.css") layer(project);
+@import url("./for-core-cms/components/django.cms.picture.css") layer(project);
```

Comment:
This change is unexpected, especially since #355. Merge conflict resolution gone awry? Please help me understand.

<details>

I do **not** see `django.cms.picture.css` imported in https://github.com/TACC/tup-ui/blob/feat/tup-463/apps/tup-cms/src/taccsite_custom/tup_cms/static/tup_cms/css/tup-cms.for-core-cms.css.

</details>